charts/project-origin-wallet/values.yaml

```diff
@@ -31,6 +31,9 @@ config:
     # audience defines the audience to use for the jwt tokens, defaults to empty string, which means the audience is not checked
     audience: ""
 
+    # enableScopeValidation determines if scope validation is enabled, defaults to false, if set to true then scope validation is enabled
+    enableScopeValidation: false
+
     # allowAnyJwtToken determines if any jwt token is allowed, defaults to false, if set to true then any jwt token is allowed, NOT RECOMMENDED FOR PRODUCTION!
     allowAnyJwtToken: false
```

Comment:
Should these 2 settings be completely separate? should one be able to enable scopes without enabling the allowAnyJwtToken ENV variable?

Reply:
allowAnyJwtToken allows the jwt to be issued by anyone, it simply accepts everythin.
enableScopeValidations enables that it the scopes are required 

the can be used independent of each other.